tests/data/validation/enum/enum.go

```diff
@@ -96,6 +106,16 @@ func (j *EnumMyBooleanUntypedEnum) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Verify checks all fields on the struct match the schema.
+func (plain EnumMyBooleanUntypedEnum) Verify() error {
+	for _, expected := range enumValues_EnumMyBooleanUntypedEnum {
+		if reflect.DeepEqual(v, expected) {
```

Comment:
where does this `v` come from? I don't see it declared anywhere, are you sure it's valid?